content/en/tracing/troubleshooting/tracer_startup_logs.md

```diff
@@ -0,0 +1,254 @@
+{"date":"2020-06-29T12:26:39.5615724+02:00","os_name":"Windows",
+"os_version":"Microsoft Windows NT 6.2.9200.0","version":"1.17.1.0",
+"platform":"x64","lang":".NET Framework","lang_version":"4.8+",
+"env":null,"enabled":true,"service":"git-credential-manager",
+"agent_url":"http://localhost:8126","debug":false,
+"analytics_enabled":false,"sample_rate":null,"sampling_rules":null,
+"tags":[],"log_injection_enabled":false,
+"runtime_metrics_enabled":false,"disabled_integrations":[]}
+```
+
+#### Diagnostics
+
+The .NET tracer has many diagnostic lines it can print.  Please see below for a complete list:
+
+```text
+DATADOG TRACER DIAGNOSTICS - Profiler disabled in DD_TRACE_ENABLED
+DATADOG TRACER DIAGNOSTICS - Profiler disabled: {process_name} not found in DD_PROFILER_PROCESSES
+DATADOG TRACER DIAGNOSTICS - Profiler disabled: {process_name} found in DD_PROFILER_EXCLUDE_PROCESSES
+DATADOG TRACER DIAGNOSTICS - Failed to attach profiler: interface ICorProfilerInfo3 not found
+DATADOG TRACER DIAGNOSTICS - Profiler disabled: {application_pool} is recognized as an Azure App Services infrastructure process
+DATADOG TRACER DIAGNOSTICS - Profiler disabled: {application_pool} is recognized as Kudu, an Azure App Services reserved process
+DATADOG TRACER DIAGNOSTICS - Profiler disabled: DD_INTEGRATIONS environment variable not set
+DATADOG TRACER DIAGNOSTICS - Profiler disabled: no enabled integrations found
+DATADOG TRACER DIAGNOSTICS - Failed to attach profiler: unable to set event mask
+DATADOG TRACER DIAGNOSTICS - Error fetching configuration {exception}
+```
+
+{{% /tab %}}
+{{% tab "PHP" %}}
+
+#### Configuration
+
+```text
+[2020-07-01T17:42:50Z] DATADOG TRACER CONFIGURATION - {"agent_error":"Couldn't connect to server","ddtrace.request_init_hook_reachable":false,"date":"2020-07-01T17:42:50Z","os_name":"Linux 49b1cb4bdd12 4.19.76-linuxkit #1 SMP Tue May 26 11:42:35 UTC 2020 x86_64","os_version":"4.19.76-linuxkit","version":"1.0.0-nightly","lang":"php","lang_version":"7.4.5","env":null,"enabled":true,"service":null,"enabled_cli":false,"agent_url":"https://localhost:8126","debug":false,"analytics_enabled":false,"sample_rate":1.000000,"sampling_rules":null,"tags":null,"service_mapping":null,"distributed_tracing_enabled":true,"priority_sampling_enabled":true,"dd_version":null,"architecture":"x86_64","sapi":"cgi-fcgi","ddtrace.request_init_hook":null,"open_basedir_configured":false,"uri_fragment_regex":null,"uri_mapping_incoming":null,"uri_mapping_outgoing":null,"auto_flush_enabled":false,"generate_root_span":true,"http_client_split_by_domain":false,"measure_compile_time":true,"report_hostname_on_root_span":false,"traced_internal_functions":null,"auto_prepend_file_configured":false,"integrations_disabled":null,"enabled_from_env":true,"opcache.file_cache":null}
+```
+
+#### Diagnostics
+
+The Diagnostics for the PHP tracer will print if the tracer is in DEBUG mode. They will be available through the other options below.
```

Comment:
```suggestion
The Diagnostics for the PHP tracer print if the tracer is in DEBUG mode. They are available through the other options below.
```

Reply:
but also, it's unclear to me what "available through the other options below" means

Reply:
PHP Info section, or CLI SAPI, etc. all the remaining sections are just ways to get the same information